channels-and-providers/email/maqsam.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Maqsam'
+description: 'Learn how to use the Maqsam provider to send sms notifications using Novu'
+---
+
+[Maqsam](https://maqsam.com/) is the friendliest cloud telephony solution for SMEs.
+
+# Getting Started
+
+To use the Maqsam provider in the sms channel, you will need to create a Maqsam account and add your access key & secret to the Maqsam integration on the Novu platform.
+Contact `support@maqsam.com` to enable the API feature for your account, then you can generate `access_key_id` and `access_secret` combination from the account settings.
+
+# Creating the Maqsam integration with Novu
+
+- Visit the [Integrations](https://web.novu.co/integrations) page on Novu.
+- Select Maqsam service
```

Comment:
Please add the step where the user needs to click on "Add a provider" before the step where the select the Maqsam provider.